### DISCUSSION TITLE
Lesson 13 - Serving Live

### DISCUSSION THREAD

**Discussion by Programproductions** [smartcontractkit/full-blockchain-solidity-course-py#299] [Q&A] [answered]
@PatrickAlphaC - first of all thanks again for a great course! It's been a real journey.  I'm about to do the additional work on Lesson 13, but before doing that I thought I would try to try it out online. 

I put it on the web and it works great on my laptop browser, but when I try to use the Dapp on a mobile it won't connect to Metamask. I have Metamask app installed on my phone, and Kovan selected, but no joy.

Is this caused by the useDapp Package? I tried their example on my phone to and it returned "No Ethereum provider found on window.ethereum"

Has anyone else tried out uploading their tutorial?

**Comment by cromewar** [comment, marked as answer]
Hello @Programproductions, to run the app on your phone you would like to follow this steps:

Even if you have installed metamask on your cellphone, if you use a browser like Chrome it would not work, however metamask app has its own web browser inside, try to use that one. 

Or you can install kiwi browser on your phone which allows you to install extensions as it was a desktop browser, then install metamask on it.

**Reply by Programproductions** [reply]
hey, @cromewar - thanks! It worked great inside my Metamask browser :)

Just wondering why chrome supports connection on a laptop browser but not a mobile?

**Reply by cromewar** [reply]
Well I think the state of the art is still immature, may be on the coming years they would have support.
